src/layout/questionnaire-nav/components/arbo-simplifield.jsx

```diff
@@ -84,16 +62,17 @@ class ArboSimplified extends Component {
     }, {});
```

Comment:
map(function, thisArg) culd be replaced with map(function) when no reference to thisArg in the function